doc/en/api/1.0.0/styles.yaml

```diff
@@ -85,36 +102,23 @@ paths:
         
         The style definition body can be requested by either appending the file extension of the style file (e.g., "{style}.sld" or "{style}.css") or by specifying the correct media type for the style definition in the "Accept" header. Below are common style formats and the corresponding media types that can be used in the Accept header to request the style definition body.
         
-        - application/vnd.ogc.sld+xml for SLDs
+        - application/vnd.ogc.sld+xml for SLD 1.0.0 SLDs
+        - application/vnd.ogc.se+xml for SLD 1.1.0 SLDs
         - application/vnd.geoserver.geocss+css for css styles
         - application/vnd.geoserver.ysld+yaml for ysld styles
         - application/vnd.geoserver.mbstyle+json for mb styles
       produces:
         - application/xml
         - application/json
         - application/vnd.ogc.sld+xml
+        - application/vnd.ogc.se+xml
         - text/html
       parameters:
         - name: style
           in: path
           required: true
           description: Name of the style to retrieve.
           type: string
-        - name: purge
-          in: query
-          required: false
-          description: Specifies whether the underlying file containing the style should be deleted on disk. Allowable values are "true" or "false" (default).
-          type: boolean
```

Comment:
what was this doing here?

Reply:
Excellent question. No idea. Seems like it came from a bulk copy-pasting of parameters.